restcomm/restcomm.rvd/src/main/webapp/languages/es.json

```diff
@@ -54,10 +55,10 @@
 	"designerValidationErrorLabel3": "ha sido programado",
 	"designerModulesToggleFilteringTitle": " Activar filtrado en despliegue de módulos",
 	"designerInputSearchLabelTitle": "Buscar módulos por etiqueta",
-	"designerInputSearchRegexTitle": "Escriba un filtro para reducir lista de módulos. Sintaxis  de Regex es compatible",
+	"designerInputSearchRegexTitle": "Escriba un filtro para reducir lista de módulos. Sintaxis Regex es compatible",
 	"designerInputSearchPlaceholder": "Etiqueta de módulo",
 	"designerInputSearchHelp": "Búsqueda de módulos",
-	"designerEditListedLinkTitle": "Lista de módulos abiertos para editar",
+	"designerEditListedLinkTitle": "Abrir módulos en lista para editar",
 	"designerEditListedLink": "Editar lista",
 	"designerHideAllLinkTitle": "Ocultar todos los módulos exceptuando el inicial",

```

Comment:
In line 49, change for:
"designerStartupModuleLinkTitle": "Seleccione el módulo con el que comenzará su aplicación",